cmd/kured/main.go

```diff
@@ -531,6 +538,31 @@ func maintainRebootRequiredMetric(nodeID string, sentinelCommand []string) {
 	}
 }
 
+func maintainRestartInProgressMetric(nodeID string) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Please use `Error` instead. Non-functional metrics should not kill kured.

Reply:
I originally triggered off of reboots but smaller nodes reboot so fast that the metric cannot get scraped before kured reboots. I figured a node lock was somewhat safe to indicate a reboot in progress was going to occur.